Packs/EmailCommunication/Scripts/PreprocessEmail/README.md

```diff
@@ -1,9 +1,13 @@
 Preprocessing script for Email Communication layout.
-This script checks incoming emails from the incident type. If those emails contain an Incident ID in the email subject, then the preprocessing will not open a new incident for this email. Instead, it will add the email response to the existing incident war room with the  "email-thread" tag.
+This script checks incoming emails from the incident type. If those emails contain an Incident Unique ID in the email subject, then the preprocessing will not open a new incident for this email. Instead, it will add the email response to the existing incident war room with the  "email-thread" tag.
+
+**Note:** In order to avoid performance issues, incoming emails will be added to an existing incident as "email-thread" only if the incident was modified in the last 2 months.
+If you wish to extend this period, you can make a copy of the script and change the `QUERY_TIME` constant in line #12 to the desired period.
```

Comment:
and configure the new script to be the pre-processing script of the incident type `Email Communication`.